src/SavedSearch.php

```diff
@@ -268,6 +268,21 @@ public function canCreateItem(): bool
         return parent::canCreateItem();
     }
 
+    /**
+     * Check if user can view saved searches
+     * Allow access in helpdesk interface for private searches
+     */
```

Comment:
No need to override the parent PHPDoc, the in-code comment already explains the behaviour.

```suggestion
```